forecast/paper/response-to-reviewers.tex

```diff
@@ -0,0 +1,304 @@
+\documentclass[11pt]{article}
+
+\usepackage[T1]{fontenc}
+\usepackage{geometry}
+\usepackage{microtype}
+
+\usepackage{newtxmath}
+\usepackage{newtxtext}
+
+
+\usepackage{xcolor}
+\definecolor{cobalt}{rgb}{0.0, 0.28, 0.67}
+
+\newenvironment{resp}{\begin{quote}\color{cobalt}}{\end{quote}}
+\newcommand{\sresp}[1]{\textcolor{cobalt}{#1}}
+
+\setlength{\parindent}{0in}
+\setlength{\parskip}{11pt}
+
+\title{Response to peer reviews}
+\author{}
+\date{\today}
+
+\begin{document}
+\maketitle
+
+
+\section*{Editor}
+
+
+  Reviews are generally supportive, but bring up several issues that need to be
+  addressed. Once addressed, please submit the revised manuscript with an
+  item-by-item reply/rebuttal.  
+
+
+\begin{resp}
+  We thank the editor and reviewers for their comments and the opportunity to make
+  revisions. We have made a number of revisions, described in detail below.
+  We believe these changes have strengthened the manuscript, and describe them in
+  more detail below.
+  We quote below the Reviewer comment and provide our response in this color.
+\end{resp}
+
+
+\section*{Reviewer 1}
+
+
+
+  1. In their presentation of of evaluation metrics they do not discuss
+  stability. For instance in [7] does the choice of the weights, using $\tau$ for
+  $x\geq 0$ rather than some monotone function say some power $\tau^\alpha$ make a
+  difference? Since as they indicate the measure is used for fitting as well as
+  ascertainment, some stability investigation seems appropriate. I suspect here
+  is no great effect but...
+
+
+\begin{resp}
+  The reviewer is correct: the choice of $\tau$ rather than a monotone function of
+  $\tau$ will make a difference. But it seems that we should have been more clear
+  in our description of WIS. The weights used here are those used to evaluate
+  forecasts submitted to the COVID-19 Forecast Hub and the CDC. More broadly,
+  this is a standard metric in the forecasting community (just as Mean Squared
+  Forecast Error is standard), rather than an ``author decision''.  In principle,
```

Comment:
According to [the WIS paper](https://arxiv.org/pdf/2005.12881.pdf), in the text between (2) and (3), this choice of weights (along with equally spaced quantile levels and K being large) approximates CRPS.  So this could be further motivation beyond it being the version used on the COVID Hub. In terms of this choice of weights for WIS being standard in the forecasting community, I don't see this mentioned in the WIS paper.  If it's true, can we cite some reference(s)?

Reply:
"Equivalent to Quantile Loss (discrete version of CRPS)."